### ISSUE TITLE
[BUG] Multiple monitors next to each other breaks some games mouse focus

### ISSUE THREAD

**Issue by Minc3** [pop-os/cosmic-epoch#1070]
If the game is in full screen or sometime in border-less after running cosmic-rander I then have to click SUPER key then click on the game window to get mouse focus.

cosmic-randr mode DP-2 3840 2160 --refresh 144 --pos-y 0 --pos-x 3840
cosmic-randr mode DP-2 3840 2160 --refresh 144 --pos-y 2160 --pos-x 3840

I also have this in my steam launch options:
SDL_VIDEO_MINIMIZE_ON_FOCUS_LOSS=0

In the video I have keyboard shortcuts setup.
https://github.com/user-attachments/assets/e3bb723c-924e-4d0a-ac3b-9f87ed0e1606

**Comment by Minc3** [comment]
Embed Video:
https://github.com/user-attachments/assets/a009c748-5a21-46cc-98fd-a082aa993d0d

**Comment by Minc3** [comment]
**Monitors next to each other: (Default Mode)**

`
DP-1 (enabled)
  Model:
  Physical Size: 630 x 360 mm
  Position: 7680,2160
  Scale: 100%
  Transform: normal
  Adaptive Sync: false

DP-2 (enabled)
  Model:
  Physical Size: 630 x 360 mm
  Position: 3840,2160
  Scale: 100%
  Transform: normal
  Adaptive Sync: false

DP-3 (enabled)
  Model:
  Physical Size: 630 x 360 mm
  Position: 0,2160
  Scale: 100%
  Transform: normal
  Adaptive Sync: false
  `
  
**Main monitor above the others:**
  
  `
DP-1 (enabled)
  Model:
  Physical Size: 630 x 360 mm
  Position: 7680,2160
  Scale: 100%
  Transform: normal
  Adaptive Sync: false

DP-2 (enabled)
  Model:
  Physical Size: 630 x 360 mm
  Position: 3840,0
  Scale: 100%
  Transform: normal
  Adaptive Sync: false

DP-3 (enabled)
  Model:
  Physical Size: 630 x 360 mm
  Position: 0,2160
  Scale: 100%
  Transform: normal
  Adaptive Sync: false

`

**Comment by jjuel** [comment]
Yeah this is an issue. Even if the game is in fullscreen mode in Steam you can just slide out of the window onto the next monitor, and that minimizes the game. Have had this issue in both Dota2 and CS2.
